src/location_backup.c

```diff
@@ -0,0 +1,36 @@
+#include "global.h"
+#include "field_system.h"
+#include "location_backup.h"
+#include "save_local_field_data.h"
+#include "constants/maps.h"
+
+static const Location _020FA17C = {
```

Comment:
this location is inside the player's house (at the start of the game)

one below is starting town, prolly in front of the house